Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -141,6 +141,10 @@ BTDiagnostics::ReadParameters ()
         m_crse_ratio == amrex::IntVect(1),
         "Only support for coarsening ratio of 1 in all directions is included for BTD\n"
         );
+    WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
+        warpx.Geom(0).ProbHi(WARPX_ZINDEX) <= 0._rt,
+        " The geometry.prob_hi of the boosted-frame domain in the moving window direction must be ==0.\n"
+        );
```

Comment:
This ASSERT is to be discussed with @RemiLehe and @atmyers 
This is what works with the way we do BTD as of now, since we don't add an offset when converting from boosted frame to lab-frame co-ordinates.

Reply:
update : discussed offline with @RemiLehe 
we are looking into the expressions used to obtain z coordinates in the lab-frame, and checking whether offsets are needed here